src/libs/humanizer/modules/Uniswap/uniUniversalRouter.ts

```diff
@@ -31,26 +31,17 @@ const extractParams = (inputsDetails: any, input: any) => {
 }
 
 // '0x1234' => ['0x12', '0x34']
-function parseCommands(commands: string, emitError?: Function): string[] | null {
-  try {
-    if (!commands.startsWith('0x') || commands.length % 2 !== 0) return null
-    const hex = commands.slice(2)
-    const hexRegex = /[0-9A-Fa-f]/g
-    if (!hexRegex.test(hex)) return null
-    const res = hex.match(/.{2}/g)?.map((p: string) => `0x${p}`)
-    return res as string[]
-  } catch (e) {
-    emitError &&
-      emitError({
-        level: 'minor',
-        message: 'Unexpected error in parsing uniswap commands',
-        error: e
-      })
-    return null
+function parseCommands(commands: string): string[] | null {
```

Comment:
I am not sure i understand if this works properly, can you leave a comment what it does?